docs/faas/aws-lambda.md

```diff
@@ -66,6 +66,10 @@ contain an incomplete trace context which indicates X-Ray isn’t enabled. The e
 `Context` will be valid and sampled only if AWS X-Ray has been enabled for the Lambda function. A user can
 disable AWS X-Ray for the function if the X-Ray Span Link is not desired.
 
+Instrumentation SHOULD include an explicit option to use `Context` in `_X_AMZN_TRACE_ID` as parent insted of `Link`,
+as long as it is a valid `Context`, as described above. Else, it will follow the standard behavior of using the extracted value
```

Comment:
```suggestion
as long as it is a valid `Context`, as described above. If the option is not present, it will follow the standard behavior of using the extracted value
```